portal-web/docroot/html/portlet/plugins_admin/edit_plugin.jsp

```diff
@@ -42,7 +42,7 @@ if (pluginType.equals(Plugin.TYPE_PORTLET)) {
 %>
 
 <portlet:actionURL var="editPluginURL">
-	<portlet:param name="struts_action" value="/plugins_admin/edit_plugin" />
+	<portlet:param name="mvcRenderCommandName" value="/plugins_admin/edit_plugin" />
```

Comment:
For actionURLs you shouldn't use mvcRenderCommandName but it should be in the name attribute of the actionURL. Something like:

```java
<portlet:actionURL name="/blogs/edit_entry" var="editEntryURL" />
```